code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -79,7 +79,6 @@
 
 	//halucination vars
 	var/image/halimage
-	var/image/halbody
 	var/obj/halitem
```

Comment:
Are these two still used?

Reply:
`halimage` no, `halitem` yes, though looking at the code `halitem` is only used in one part--I can get rid of it.